iModelCore/ECDb/ECDb/ECSql/ECSqlParser.cpp

```diff
@@ -5,6 +5,7 @@
 #include "ECDbPch.h"
 #include "Parser/SqlNode.h"
 #include "Parser/SqlParse.h"
+#include "ECSqlParser.h"

```

Comment:
This include should not be needed since it's part of the Pch

Reply:
removed